test/lint.js

```diff
@@ -8,4 +8,12 @@ var blint = require("blint");
   });
 });
 
+["src"].forEach(function(dir) {
+  blint.checkDir(dir, {
+    browser: true,
+    blob: "// CodeMirror, copyright (c) by Marijn Haverbeke and others\n// Distributed under an MIT license: http:\/\/codemirror.net\/LICENSE\n\n",
```

Comment:
(I'm happy to get rid of the per-file header thing. If that's in the way at any point, feel free to just drop it. I should probably have mentioned that earlier.)